src/routes/auth.route.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const { authController } = require('../controllers/auth.controller');
+const { authMiddleware } = require('../middlewares/authMiddleware');
+
+const router = express.Router();
+
+router.post('/signup', authController.authValidation, authController.signUp);
+router.post('/signin', authController.authValidation, authController.signIn);
```

Comment:
Ensure that `authValidation` is correctly exported and available in the `authController`. If it's not part of the `authController`, it should be imported separately.